src/openms/source/FORMAT/MSstatsFile.cpp

```diff
@@ -497,11 +508,14 @@ void OpenMS::MSstatsFile::storeISO(const String& filename,
   map< pair< String, unsigned >, unsigned> path_label_to_fractiongroup = design.getPathLabelToFractionGroupMapping(true);
 
   // The Retention Time is additionally written to the output as soon as the user wants to resolve multiple peptides manually
-  const bool rt_summarization_manual(retention_time_summarization_method == "manual");
+  bool rt_summarization_manual(retention_time_summarization_method == "manual");
 
-  if (rt_summarization_manual)
+  if (!rt_summarization_manual)
   {
-    cout << "WARNING: One feature might appear at multiple retention times in the output file. This is invalid input for MSstats. Combining of features over retention times is needed!" << endl;
+    OPENMS_LOG_WARN << "WARNING: rt_summarization set to something else thatn 'manual' but MSstatsTMT does aggregation of"
+            " intensities of peptide-chargestate combinations in the same file itself."
+            " Reverting to 'manual'" << endl;
+    rt_summarization_manual = true;
```

Comment:
should we add the possibility to force a different aggregation method, when using the MSstatsConverter with TMT data? E.g. if someone wants to use it for aggregation and work with his own code. Or should that not be intended?

Reply:
hmm don't know if that is a relevant use case. It implies that you don't use MSstats and I guess you would just work with the raw data and could do the max / min etc. calculations in the scripting language yourself.

Reply:
Yes, seems fair! Let's leave it at that.